scio-parquet/src/main/scala/com/spotify/scio/parquet/read/ParquetRead.scala

```diff
@@ -43,8 +58,92 @@ object ParquetRead {
     readSupportFactory: ReadSupportFactory[T],
     conf: SerializableConfiguration,
     projectionFn: SerializableFunction[T, R]
-  ): ParDo.SingleOutput[ReadableFile, R] = {
+  ): PTransform[PCollection[ReadableFile], PCollection[R]] = {
     val sdf = new ParquetReadFn[T, R](readSupportFactory, conf, projectionFn)
-    ParDo.of(sdf)
+    val tfx: PTransform[PCollection[_ <: ReadableFile], PCollection[R]] = ParDo.of(sdf)
+
+    tfx.asInstanceOf[PTransform[PCollection[ReadableFile], PCollection[R]]]
+  }
+
+  def readTyped[T: ClassTag: Coder: ParquetType](
+    predicate: FilterPredicate = null,
+    conf: Configuration = null
+  ): PTransform[PCollection[ReadableFile], PCollection[T]] = readTyped[T, T](
+    identity,
+    predicate,
+    conf
+  )
+
+  def readTyped[T: ClassTag: Coder: ParquetType, R](
+    projectionFn: T => R,
```

Comment:
Why 2 APIs and not setting `identity` as default projectionFn ?

Reply:
Just to make the API friendlier for users who aren't supplying a projection, so they con't have to specify their `T` type twice